scripts/measurements/darshini-test.sh

```diff
@@ -10,10 +10,10 @@
 
 truncate -s 0 darshini-test.log
 
-echo "performance of Darshini on four processors"  >> darshini-test.log
+
 echo "--------------------------------------" >> darshini-test.log
 
-for pcapFile in `ls Test_pcap_files/sample_capture_test*.pcap`
+for pcapFile in `sudo ls <Path to BITS-Darshini>/data/packet/*.pcap`
```

Comment:
sudo not needed here. We can give a relative path name here (same way the path way provided in tools_run.sh script).

Reply:
@prasadtalasila Relative path name here is not working as we have to give the path name to the analyzer in BITS-Darshini which requires the whole path name. I think that we can maybe add pwd command in the script and store it result in a variable and use that in ls command.